applications/storage/storage.c

```diff
@@ -31,16 +31,20 @@ static void storage_app_sd_icon_draw_callback(Canvas* canvas, void* context) {
 
 Storage* storage_app_alloc() {
     Storage* app = malloc(sizeof(Storage));
+    memset(app, 0, sizeof(Storage));
```

Comment:
malloc is already doing it

Reply:
no guarantees from specification, though

Reply:
guaranteed on our side )